packages/project/src/v4-specification.ts

```diff
@@ -406,7 +406,7 @@ export interface Project {
    * }
    *
    * @minItems 1
-   * @maxItems 25
+   * @maxItems 200
```

Comment:
Will even this be enough for upcoming partnerships? This isn't enforced in any way, right? It's just documentation

Reply:
Correct, just for docs at this stage.